VoucherifyClient.php

```diff
@@ -143,6 +143,25 @@ public function redeem($code, $trackingId) {
         }
 
         /**
+         * @param string|array $code Voucher code or array with voucher (code), customer profile and order amount
+         * @param string|null $trackingId Provided tracking id
+         *
+         * Validate voucher
+         *
+         * @throws Voucherify\ClientException
+         */
+        public function validate($code, $trackingId = NULL) {
```

Comment:
validate function signature should be `validate ($code, $context)` where code is a string and context is an array or object.